src/core.js

```diff
@@ -309,6 +309,13 @@ $.extend( $.validator, {
 				this.element( element.parentNode );
 			}
 		},
+		onchange: function( element, event ) {
```

Comment:
Wouldn't that be the same for any of the event handlers though? Being able to override and lose the default behavior unless set up in the override?

If that's an issue, another option, I can throw the logic into delegate, but originally I avoided that to prevent throwing in specific event logic into that function.

Or any other ideas?